### ISSUE TITLE
Possible regression from 2.4.0.2 to 2.5.0.0? Parser::SyntaxError: unexpected token kDO

### ISSUE THREAD

**Issue by jaredbeck** [whitequark/parser#451]
- 2.5.0.0 produces `Parser::SyntaxError: unexpected token kDO`
- 2.4.0.2 does not

### Reproduction

```ruby
require "bundler/inline"

gemfile(true) do
  ruby "2.4.3"
  source "https://rubygems.org"
  # gem "parser", "2.4.0.2"
  gem "parser", "2.5.0.0"
end

require 'parser/ruby24'
Parser::Ruby24.parse(
  <<~EOS
    def b
      c [:d], ::E::F do
        # block contents irrelevant
      end
    end
  EOS
)
```

**Comment by iliabylich** [comment]
Yes, it was fixed in https://github.com/whitequark/parser/pull/448, wait for the next release please.
``` sh
$ cat test.rb
def b
  c [:d], ::E::F do
    # block contents irrelevant
  end
end

$ bin/ruby-parse test.rb
(def :b
  (args)
  (block
    (send nil :c
      (array
        (sym :d))
      (const
        (const
          (cbase) :E) :F))
    (args) nil))
```

**Comment by jaredbeck** [comment]
> Yes, it was fixed in #448, wait for the next release please.

Thanks Ilya! :heart:
